packages/sit-onyx/src/components/OnyxDataGrid/features/sorting/types.ts

```diff
@@ -0,0 +1,26 @@
+import { type MaybeRef, type MaybeRefOrGetter } from "vue";
+import type { DataGridEntry } from "../../types";
+
+export type SortDirection = "asc" | "desc" | "none";
+
+/**
+ * -1: less than, 0: equal to, 1: greater than
+ */
+export type Compare<T> = (a: T, b: T) => -1 | 0 | 1;
```

Comment:
Do we really want to type it that strict? The regular `.sort()` method supports all negative/positive numbers so its easier to do things like `price1 - price2` to sort by numeric values